annotation-utils/src/main/java/edu/isi/vista/annotationutils/ExportAnnotations.kt

```diff
@@ -150,11 +151,12 @@ fun main(argv: Array<String>) {
                         document.name
                     }
 
-                    val jsonBytes = it.getInputStream("$zipEntryName.json")?.readAllBytes()
-
+                    val jsonBytes = it.getInputStream("$zipEntryName.json")?.readBytes()
                     if (jsonBytes != null) {
-                        val outFileName = documentOutputDir.resolve("${document.name}-${annotationRecord.user}.json");
-                        Files.write(outFileName, jsonBytes)
+                        var jsonTree = ObjectMapper().readTree(jsonBytes) as ObjectNode
+                        jsonTree.replaceFieldEverywhere("sofaString", "__DOCUMENT_TEXT_REDACTED_FOR_IP_REASONS__")
```

Comment:
Add a comment about why we do this (that we cannot redistribute LDC documents and the user can always restore from those LDC documents)